addons/mail/static/src/models/composer/composer.js

```diff
@@ -176,7 +176,7 @@ function factory(dependencies) {
             const recipients = [...this.mentionedPartners];
             if (this.thread) {
                 for (const recipient of this.thread.suggestedRecipientInfoList) {
-                    if (recipient.partner && recipient.isSelected) {
+                    if (recipient.partner && recipient.isSelected && !this.isLog) {
```

Comment:
Move up the condition on isLog at at line 177, no need to iterate for nothing

Reply:
True, I'm stupid...